src/Bridge/Doctrine/Orm/Filter/RangeFilter.php

```diff
@@ -102,7 +101,7 @@ private function addWhere(QueryBuilder $queryBuilder, QueryNameGeneratorInterfac
                 $rangeValue = explode('..', $value);
 
                 if (2 !== count($rangeValue)) {
```

Comment:
Filtering on invalid properties does not throw exceptions, does it? Yes, I'd love to ease debugging as well, but we really should be consistent.